src/ui/spoiler_card/spoilerCardActions.js

```diff
@@ -1,23 +1,48 @@
 import {action} from "mobx";
 import MainStore from "../MainStore";
+import ToastStore from "../toast/ToastStore";
+import addActions from "../add_card/addActions";
 
 // TODO: ES2015
 module.exports = {
+	isValidTitle: action(isValidTitle),
+	editTitle: action(editTitle),
 	editTags: action(editTags)
 };
 
+function isValidTitle(title) {
+	if (addActions.isInvalidTitle(title)) {
+		ToastStore.isInvalidTitleOrTags = true;
+		return false;
+	}
+	else if (addActions.isDuplicateTitle(title)) {
+		ToastStore.isDuplicateTitle = true;
+		return false;
+	} else {
+		return true;
+	}
+}

```

Comment:
I'd like not to introduce a dependency on addActions. Do you think shifting isInvalidTitle, isDuplicateTitle to a commonActions file makes more sense?